audit/projects/k8s-infra-e2e-boskos-scale-01/iam.json

```diff
@@ -30,8 +30,7 @@
     {
       "members": [
         "group:k8s-infra-prow-oncall@kubernetes.io",
-        "group:k8s-infra-sig-scalability-oncall@kubernetes.io",
-        "user:spiffxp@google.com"
+        "group:k8s-infra-sig-scalability-oncall@kubernetes.io"
```

Comment:
Expected, this me, same thing as https://github.com/kubernetes/k8s.io/pull/1800#discussion_r604577472